libs/@guardian/source-react-components/src/choice-card/theme.ts

```diff
@@ -1,6 +1,21 @@
 import { palette } from '@guardian/source-foundations';
 import { userFeedbackThemeDefault } from '../user-feedback/theme';
 
+export interface ChoiceCardTheme {
+	textUnSelected?: string;
+	textSelected?: string;
+	textHover?: string;
+	textError?: string;
+	borderUnSelected?: string;
+	borderSelected?: string;
+	borderHover?: string;
+	borderError?: string;
+	backgroundUnSelected?: string;
+	backgroundHover?: string;
+	backgroundSelected?: string;
+	backgroundTick?: string;
+}
+/** @deprecated use ChoiceCardThemeLight **/
```

Comment:
Should the deprecation warning say something about not using `ThemeProvider` too as we want people to use the `theme` prop rather than simply swapping themes?

```suggestion
/** @deprecated Use `choiceCardThemeLight` and `theme` prop instead of `ThemeProvider` **/
```